mrmustard/physics/representations.py

```diff
@@ -523,8 +555,9 @@ class Fock(Representation):
 
     """
 
-    def __init__(self, array: Batch[Tensor], batched=False):
+    def __init__(self, array: Batch[Tensor], batched=False, original_bargmann_data=None):
```

Comment:
Is Fock user-facing? How do we ensure that the array given is consistent with original_bargmann_data?

Reply:
How do mean consistent? It cannot come from anything else, if that's what you meant

Reply:
Oh, I see the question now. Yes, this could be more privately handled.